x-pack/plugins/fleet/common/types/models/agent.ts

```diff
@@ -36,8 +36,7 @@ export type AgentActionType =
   | 'UNENROLL'
   | 'UPGRADE'
   | 'SETTINGS'
-  // INTERNAL* actions are mean to interupt long polling calls these actions will not be distributed to the agent
-  | 'INTERNAL_POLICY_REASSIGN';
+  | 'POLICY_REASSIGN';
```

Comment:
do you think we would need to add a migration script for existing `fleet-agent-actions` SOs to transform `INTERNAL_POLICY_REASSIGN` to `POLICY_REASSIGN`?

Reply:
I think it's okay actions are not mean to stay for a long time, so I guess we are okay moreover with the move to fleet server we should start from a fresh ground and do not have any agent already enrolled.